lectures/heavy_tails.md

```diff
@@ -11,6 +11,8 @@ kernelspec:
   name: python3
 ---
 
++++ {"user_expressions": []}
+
```

Comment:
Please remove this tag and the tags below.

I usually use a global replacement to remove them in on click : )